src/Umbraco.Web.UI.Client/src/views/propertyeditors/markdowneditor/markdowneditor.controller.js

```diff
@@ -27,6 +27,19 @@ function MarkdownEditorController($scope, $element, assetsService, editorService
         editorService.mediaPicker(mediaPicker);
     }
 
+    function openLinkPicker(callback) {
+        var linkPicker = {
```

Comment:
@kjac While not documented, the link picker editor already has a `hideTarget` property - I'm using it in my redirects package here:

https://github.com/skybrud/Skybrud.Umbraco.Redirects/blob/7f8680784de6808fa60eb707e2c1385d5e18a22d/src/Skybrud.Umbraco.Redirects/App_Plugins/Skybrud.Umbraco.Redirects/Scripts/Controllers/Editors/Destination.js#L6

Reply:
@abjerner nice! I'll get that fixed asap.

Reply:
Works brilliantly.. all done!